cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractStorageProviderNamespaceAdmin.java

```diff
@@ -145,14 +144,6 @@ private boolean hasCustomLocation(NamespaceMeta namespaceMeta) {
   }
 
   private void validateCustomLocation(NamespaceMeta namespaceMeta) throws IOException {
-    // a custom location was provided
-    // check that its an absolute path
-    File customLocation = new File(namespaceMeta.getConfig().getRootDirectory());
-    if (!customLocation.isAbsolute()) {
-      throw new IOException(String.format(
-        "Cannot create the namespace '%s' with the given custom location %s. Custom location must be absolute path.",
-        namespaceMeta.getName(), customLocation));
-    }
     // since this is a custom location we expect it to exist. Get the custom location for the namespace from
```

Comment:
we expect it to exist, but we should also check that it is a directory. I don't think that check is currently happening.